assets/src/common/helpers/test/getVideoBytesPerSecond.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This is mainly moved from `stories-editor/helpers/test/getVideoBytesPerSecond.js`, without many changes. Likewise for `isVideoSizeExcessive.js`.